javascript/builtins/Error.json

```diff
@@ -105,6 +105,56 @@
             }
           }
         },
+        "cause": {
```

Comment:
What is this feature? There's no description, URL, or spec URL. I don't know how to review this without some context.

Reply:
https://github.com/tc39/proposal-error-cause